pkg/generators/markers_test.go

```diff
@@ -0,0 +1,156 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package generators_test
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"k8s.io/kube-openapi/pkg/generators"
+)
+
+func TestParseCommentTags(t *testing.T) {
+	cases := []struct {
+		name     string
+		comments []string
+		expected generators.CommentTags
+
+		// regex pattern matching the error, or empty string/unset if no error
+		// is expected
+		expectedError string
+	}{
+		{
+			name: "basic example",
+			comments: []string{
+				"comment",
+				"another + comment",
+				"+minimum=10.0",
+				"+maximum=20.0",
+				"+minLength=20",
+				"+maxLength=30",
+				`+pattern="asdf"`,
+				"+multipleOf=1.0",
+				"+minItems=1",
+				"+maxItems=2",
+				"+uniqueItems=true",
+				"not+Minimum=0.0",
+				// "+",
```

Comment:
Would like to see a test case showing what happens when `+` is used

Reply:
added a test for +